### ISSUE TITLE
A11y_AzureToolsStorageExplorer_ToggleExplorerPaneTreeView_Keyboard: Keyboard users not able to Open Context menus using shortcut key "Shift+F10".

### ISSUE THREAD

**Issue by Suraj1200499** [microsoft/AzureStorageExplorer#2309]
GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS2.1.1; #Keyboard; #Mac; #CELAAssessID_CosmosDB-FY20-Q2;   

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.11.0
MAC Version: Catalina 10.15 

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to the blob containers section of a subscription using arrow keys.
5. No context menu opens after pressing "Shift+F10" key.

**​Actual:​**
Keyboard users not able to Open Context menus using shortcut key "Shift+F10".

Same issue is observed for **Voice Over** also.

**Expected:**
By Pressing "Shift+F10" key should open a context menu for the controls present under Explorer Pane Tree View section.

**UserImpact:**
Keyboard users and VoiceOver users are unable to reliably complete actions that require accessing the context menu, because it is not obvious how to open it from the keyboard.

**Comment by MRayermannMSFT** [comment]
Jason has merged a fix into master.

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by Suraj1200499** [comment]
@MRayermannMSFT Verified this issue in Microsoft Azure Tools Storage Explorer Application Version: 1.12.0 in MAC Version: Catalina 10.15, this issue is still reproduced.

**Comment by JasonYeMSFT** [comment]
@Suraj1200499 I have checked in a fix for it. Can you verify it on the latest build?

**Comment by Suraj1200499** [comment]
@JasonYeMSFT Verified this issue in Microsoft Azure Tools Storage Explorer Application Version: 1.12.0 in MAC Version: 10.15.3, we are still able to repro the issue for the controls present under Toggle Explorer Tree View Section.

Added Repro video to One drive link with name "2309_ContextMenu_StillReproduced". Please refer the video for further reference.

**Comment by Suraj1200499** [comment]
@MRayermannMSFT Verified this issue in Private build of Mac.
Application version: 1.12.0. 
Now by using "Shift+F10" shortcut key Context menus are accessible.
This issue is no longer reproduced. Hence Closing the bug.

Added Closed Video in [One Drive](https://microsoft-my.sharepoint.com/personal/v-nisaho_microsoft_com/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fv%2Dnisaho%5Fmicrosoft%5Fcom%2FDocuments%2FStorageExplorer%5FReprovideos&originalPath=aHR0cHM6Ly9taWNyb3NvZnQtbXkuc2hhcmVwb2ludC5jb20vOmY6L3Avdi1uaXNhaG8vRXNTVUtnZm9PSzlIbUVMOFBxcVhkOTRCVXRBVkJ6S0pldHB5SDZwS0FzVXh5dz9ydGltZT1GRkRuQ25QRzEwZw) for reference.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Mac-Sep2019;
